.github/assets/check_rv32imac.sh

```diff
@@ -7,6 +7,7 @@ crates_to_check=(
     reth-primitives-traits
     reth-network-peers
     reth-trie-common
+    reth-trie-sparse
```

Comment:
reth already guarantees that certain crates should compile under no-std riscv, so just adding this line here